src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -61,19 +66,26 @@ public async Task<IActionResult> CreateAsync([FromBody] Models.Dispute.NoticeOfD
     /// <summary>
     /// An endpoint for resending an email to a Disputant.
     /// </summary>
-    /// <param name="uuid"></param>
+    /// <param name="uuidHash"></param>
     /// <param name="cancellationToken"></param>
     /// <returns>
     /// <response code="202">Resend email acknowledged.</response>
     /// <response code="400">The uuid doesn't appear to be a valid UUID.</response>
     /// <response code="500">There was a internal server error when triggering an email to resend.</response>
     /// </returns>
-    [HttpPut("/api/disputes/email/{uuid}/resend")]
-    [ProducesResponseType(StatusCodes.Status202Accepted)]
-    [ProducesResponseType(StatusCodes.Status400BadRequest)]
-    public async Task<IActionResult> ResendEmailAsync(Guid uuid, CancellationToken cancellationToken)
+    [HttpPut("/api/disputes/email/{uuidHash}/resend")]
+    [ProducesResponseType((int)HttpStatusCode.Accepted)]
+    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    public async Task<IActionResult> ResendEmailAsync(string uuidHash, CancellationToken cancellationToken)
     {
-        EmailVerificationSend emailVerificationSend = new(uuid);
+        int[] bytes = _hashidsService.GetHashids().Decode(uuidHash);
+        string uuid = "";
+        foreach (var singleByte in bytes)
+        {
+            var singleChar = Convert.ToChar(singleByte);
+            uuid = uuid + singleChar;
+        }

```

Comment:
I might recommend this code block be moved to a static util method so it can be reused elsewhere.